docs/EnterpriseScaleAnalytics-AzureDevOpsDeployment.md

```diff
@@ -77,10 +77,14 @@ To begin, please open the [infra/params.dev.json](/infra/params.dev.json). In th
 | tags | Specifies the tags that you want to apply to all resources. | {`key`: `value`} |
 | administratorPassword | Specifies the administrator password of the sql servers. Will be automatically set in the workflow. **Leave this value as is.** | `<your-secure-password>` |
 | synapseDefaultStorageAccountFileSystemId | Specifies the resource ID of the default storage account file system for synapse. | `/subscriptions/{subscription-id}/resourceGroups/{rg-name}/providers/Microsoft.Storage/storageAccounts/{storage-name}/blobServices/default/containers/{container-name}` |
+| enableSqlPool | Specifies whether an Azure SQL Pool should be deployed inside your Synapse workspace as part of the template. If you selected dataFactory as processingService, leave this value as is. | `true` or `false` |
+| enableSqlServer | Specifies whether Azure SQL Server should be deployed as part of the template. | `true` or `false` |
+| enableCosmos | Specifies whether Azure Cosmos DB should be deployed as part of the template. | `true` or `false` |
+| enableStreamAnalytics | Specifies whether Azure Stream Analytics Cluster and Job should be deployed as part of the template. | `true` or `false` |
```

Comment:
`enableStreamAnalytics`

Reply:
Let's get this done as part of another PR.